exporter/src/main/as/flump/export/EditFormatsWindow.mxml

```diff
@@ -41,20 +41,6 @@
         <s:GridColumn dataField="scaleFactorsString" headerText="Scale Factors"/>
         <s:GridColumn dataField="textureBorder" headerText="Texture Border"/>
         <s:GridColumn dataField="maxAtlasSize" headerText="Max Atlas Size"/>
-        <s:GridColumn dataField="optimize" headerText="Optimize">
-          <s:itemEditor>
-            <fx:Component>
-              <s:ComboBoxGridItemEditor>
-                <s:dataProvider>
-                  <s:ArrayList>
-                    <fx:String>Memory</fx:String>
-                    <fx:String>Speed</fx:String>
-                  </s:ArrayList>
-                </s:dataProvider>
-              </s:ComboBoxGridItemEditor>
-            </fx:Component>
-          </s:itemEditor>
-        </s:GridColumn>
```

Comment:
I'm not familiar with MaxRects, so maybe I'm off base here, but the reason we had this setting is because (especially on mobile), it is often important to optimize to one large texture (to avoid texture swaps in a complicated scene) vs several small textures (because the alternate large texture wastes a bunch of memory). I've had to employ both settings in projects, and without seeing an example I'm not convinced that this trade-off can be tossed out the window. Your thoughts?